src/components/FakeCollectionBox.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
```

Comment:
Thank you for creating the fake collection box. This makes complete sense and I can see how this keeps viewers from being able to make changes to collections when they browse.

Reply:
You're very welcome! I'm glad it turned out alright!